ee/codegen/src/generators/nodes/bases/base.ts

```diff
@@ -294,6 +294,17 @@ export abstract class BaseNode<
         continue;
       }
 
+      // Filter out attributes that match their default values
+      const filteredAttributes = adornment.attributes.filter((attr) => {
+        if (adornment.base.name === "RetryNode") {
```

Comment:
[nit] let's create a mapping in `ee/codegen/src/generators/nodes/defaults.ts` or `ee/codegen/src/generators/nodes/constants.ts` where we start tracking these